Makefile

```diff
@@ -642,10 +637,74 @@ release-notes: $(RELEASE_NOTES_DIR) $(RELEASE_NOTES)
 promote-images: $(KPROMO)
 	$(KPROMO) pr --project cluster-api --tag $(RELEASE_TAG) --reviewers "$(IMAGE_REVIEWERS)" --fork $(USER_FORK)
 
+## --------------------------------------
+## Docker
+## --------------------------------------
+
+.PHONY: docker-pull-prerequisites
+docker-pull-prerequisites:
+	docker pull docker.io/docker/dockerfile:1.1-experimental
+	docker pull $(GO_CONTAINER_IMAGE)
+	docker pull gcr.io/distroless/static:latest
```

Comment:
Should this be moved up into build given that it is used by docker-build?

Reply:
Makes sense, done.